src/main/java/com/xiaomi/infra/pegasus/client/PegasusClient.java

```diff
@@ -72,11 +73,11 @@ private PegasusTable getTable(String tableName, int backupRequestDelayMs) throws
   // pegasus client configuration keys
   public static final String[] PEGASUS_CLIENT_CONFIG_KEYS =
       new String[] {
-        Cluster.PEGASUS_META_SERVERS_KEY,
-        Cluster.PEGASUS_OPERATION_TIMEOUT_KEY,
-        Cluster.PEGASUS_ASYNC_WORKERS_KEY,
-        Cluster.PEGASUS_ENABLE_PERF_COUNTER_KEY,
-        Cluster.PEGASUS_PERF_COUNTER_TAGS_KEY,
+        ClusterOptions.PEGASUS_META_SERVERS_KEY,
+        ClusterOptions.PEGASUS_OPERATION_TIMEOUT_KEY,
```

Comment:
I think we can implement a interferce for ClusterOptions to get client config keys.

Reply:
I can't see it makes any better. Leave it later.

Reply:
there is no need to change PegasusClient.java if we add a option to ClusterOptions.

Reply:
OK, sounds reasonable.